package.json

```diff
@@ -136,14 +136,15 @@
     "core-js": "^3.1.4",
     "enzyme": "^3.11.0",
     "enzyme-adapter-react-16": "^1.15.2",
-    "eslint": "6.6.0",
+    "eslint-plugin-jest": "^24.3.6",
     "eslint-plugin-jest-dom": "^3.6.5",
     "eslint-plugin-react": "latest",
     "eslint-plugin-react-hooks": "^2.5.0",
     "eslint-plugin-testing-library": "^3.10.2",
     "identity-obj-proxy": "^3.0.0",
     "jest-mock-promise": "^1.1.10",
     "jest-when": "^3.2.1",
+    "msw": "^0.29.0",
```

Comment:
Not used yet, but I'd like to start using it soon so I didn't remove it from package.json